src/commands/rankup.ts

```diff
@@ -208,5 +231,8 @@ See <https://docs.codewars.com/languages/> to find the correct ID.`);
     });
 
   // Format results and send
-  return send(formatResult(username, rankNums, nextRank, mode, opts.language));
-}
+  await interaction.reply({
+    content: formatResult(username, rankNums, nextRank, mode as Mode, language),
+    ephemeral: true,
```

Comment:
Do we want the result to be ephemeral?

Reply:
I think it would be good to keep the output of `/rankup` ephemeral, at least by default, since otherwise we'd have to restrict it to a dedicated channel.

Reply:
Huh, I missed this is no longer restricted. What's wrong with restricting?

I thought users enjoyed the outputs of the command triggered by others, or wanted to keep a record. I think it's better to keep the current behavior for now.

That said, allowing ephemeral result is a good new feature. Some might be curious, but not want to share the result. We can allow `/rankup` anywhere as long as it's ephemeral.